### PR TITLE
Feature/get one

### PR DESCRIPTION
**Pull request by ilija-savic-devtech**
# What's the purpose of this MR?
Get entry by id from database
# Impact Analysis
(N/A)
# Where should the tester start?
- Run start_server.bat
- Start browser and go to localhost/{id} (where {id} represents integer number)
# How should this be manually tested?
(N/A)
# Any background context you want to provide?
(N/A)
# What are the relevant tickets?
(N/A)
# Screenshots (if appropriate)
(N/A)

### CODE REVIEW

**Comment by ilija-savic-devtech** [comment]
Check it out... It's not perfect, but still working :)

**Comment by goran-erhartic-devtech** [comment]
Yo,

- do not throw exceptions in index.php, throw them before - the moment you see that you have not returned the wanted result

- put your code in try-catch blocks when there is a chance that something might go wrong (eg. contacting the DB; retrieving something from DB; etc...)

- there is no need for 'set_exception_handler()', you can just catch an error and simply display the message.. the may you have handled exceptions when contacting DBs was good, why did you change that?

**Comment by ilija-savic-devtech** [comment]
Check it out, is it ready for merging?

File: database/ServiceMongoDb.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			$object = new Student();
+			foreach ($rows as $row) {
+				$object
+					->setId($row->_id)
+					->setName($row->name)
+					->setSurname($row->surname)
+					->setIndexNo($row->indexno)
+					->setAddress($row->address);
+			}
+			$this->conn = null;
```
Comment (goran-erhartic-devtech) [inline database/ServiceMongoDb.php]: we do not need these... there are couple more of these 'connection nullifiers' so just remove those as well :)

File: database/ServiceMongoDb.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+use exceptions\InvalidIdException;
+use MongoDB\Driver\Exception\ConnectionTimeoutException;
+use MongoDB\Driver\Query;
+use src\Student;
+
+class ServiceMongoDb implements ServiceInterface
+{
+	private $conn;
+
+	public function __construct($conn)
```
Comment (goran-erhartic-devtech) [inline database/ServiceMongoDb.php]: typehint what type is $conn (as PHP is not strongly typed, this is not necessary, but highly recommended)

File: database/ServiceMySql.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+use exceptions\EmptyTableException;
+use exceptions\InvalidIdException;
+use src\Student;
+
+class ServiceMySql implements ServiceInterface
+{
+	private $conn;
+
+	public function __construct($conn)
```
Comment (goran-erhartic-devtech) [inline database/ServiceMySql.php]: typehint what type is $conn (as PHP is not strongly typed, this is not necessary, but highly recommended)

**Comment by goran-erhartic-devtech** [comment]
nice

**Pull request merged**